packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -158,6 +160,17 @@ export default class CallDiagnosticLatencies extends WebexPlugin {
     );
   }
 
+  /**
+   * Device Register Time
+   * @returns - latency
+   */
+  public getRegisterWDMDeviceJMT() {
```

Comment:
question: what does the "WDM" mean?

Reply:
getDeviceRegisterTime might be a nicer name

Reply:
We've followed the pattern of calling the function same as the JMT type in the event dictionary and prefixed with get. That's why it's called [registerWDMDeviceJMT](https://sqbu-github.cisco.com/WebExSquared/event-dictionary/blob/main/types/JMT.raml#L260)

Reply:
WDM = Webex Device Management